src/services/rpc-provider/rpc-provider.service.spec.ts

```diff
@@ -14,8 +13,8 @@ jest.mock('@ethersproject/providers', () => {
 });
 
 describe('RPC provider service', () => {
-  const MockedJsonRpcProvider = mocked(JsonRpcProvider, true);
-  const MockedWebSocketProvider = mocked(WebSocketProvider, true);
+  const MockedJsonRpcProvider = jest.mocked(JsonRpcProvider, true);
+  const MockedWebSocketProvider = jest.mocked(WebSocketProvider, true);
```

Comment:
See https://github.com/facebook/jest/pull/12089
Using mocked from utils is deprecated